Sources/StreamChat/StateLayer/MemberState.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// An observable object for the specified channel member.
+///
+/// - Note: All the properties of the represented ``ChatChannelMember`` can be accessed directly on the ``MemberState`` object (e.g. `state.name` returns ``ChatChannelMember.name``).
+@available(iOS 13.0, *)
+@dynamicMemberLookup
+public final class MemberState: ObservableObject {
```

Comment:
Why do we need this one?

Reply:
On the second thought, not really needed, removed it.